en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -242,24 +239,20 @@ $gridField->setConfig($config);
 
 ### `GridFieldConfig_RecordEditor`
 
+> [!CAUTION]
+> Permission control for editing and deleting the record uses the `canEdit()` and `canDelete()` methods on the class that represents your data.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.